lib/modules/subredditManager.js

```diff
@@ -1263,6 +1292,12 @@ async function populateSubredditDropdown(sortBy = 'subreddit', badJSON) {
 	$srList.append(theBody);
 }
 
+export function getMultiCounts(displayName) {
```

Comment:
This will become async as a result, which will be marginally annoying on line 210 but shouldn't be a huge deal (just make `manageSubreddits` async)